concepts/fluid-balance/adjusted-urine-output/CreateICUSTAY_IDarray.sql

```diff
@@ -0,0 +1,4 @@
+CREATE OR REPLACE TABLE `Tables.ICUSTAY_IDs` 
```

Comment:
I don't think this table is necessary. You can reference it as an in-line view in the AllUrineOutputAdjusted query.